backend/core/configuration.py

```diff
@@ -512,6 +528,65 @@ def handle_hijack_learn_rule_request(self, message):
                     priority=4,
                 )
 
+        def handle_load_as_sets(self, message):
+            """
+            Receives a "load-as-sets" message, translates the corresponding
+            as anchors into lists, and rewrites the configuration
+            :param message:
+            :return:
+            """
+            with open(self.file, "r") as f:
+                raw = f.read()
+            yaml_conf = ruamel.yaml.load(
+                raw, Loader=ruamel.yaml.RoundTripLoader, preserve_quotes=True
+            )
+            load_made = False
+            error = False
+            if "asns" in yaml_conf:
+                for name in yaml_conf["asns"]:
+                    if translate_as_set(name, just_match=True):
+                        ret_dict = translate_as_set(name, just_match=False)
+                        if ret_dict["ok"]:
+                            load_made = True
+                            new_as_set_cseq = ruamel.yaml.comments.CommentedSeq()
+                            for asn in ret_dict["data"]:
+                                new_as_set_cseq.append(asn)
+                            new_as_set_cseq.yaml_set_anchor(name)
+                            update_aliased_list(
+                                yaml_conf, yaml_conf["asns"][name], new_as_set_cseq
+                            )
+                        else:
+                            error = ret_dict["data"]
+                            break
+            # the as-set resolution stage failed
+            if error:
+                self.producer.publish(
+                    {"ok": False, "data": error},
+                    exchange="",
+                    routing_key=message.properties["reply_to"],
+                    correlation_id=message.properties["correlation_id"],
+                    serializer="json",
+                    retry=True,
+                    priority=4,
+                )
+            else:
+                self.producer.publish(
+                    {"ok": True, "data": None},
```

Comment:
`ok` might not be the base name for the key :P 
I know this is minor but we can use something from [json specifications](https://stackoverflow.com/questions/12806386/standard-json-api-response-format)

Reply:
So you mean:
```
{
  "success": false,
  "payload": {
    /* Application-specific data would go here. */
  },
  "error": {
    "code": 123,
    "message": "An error occurred!"
  }
}
```

Reply:
Fixed in latest iteration (note that I simply return a string for the error, we do not have pre-defined codes and I think this will complicate things at this point. I follow success, payload and error keys (gradually we need to do such refactoring in all places in the code :) ).

Reply:
Yes this seems to be covering both of our casses